deepfence_ui/app/scripts/components/common/detail-modal/header.jsx

```diff
@@ -10,7 +10,7 @@ const HeaderKVPair = ({ dataKey, value, valueAsText }) => {
   return (
     <div className={styles.alertSummaryItem}>
       <div className={styles.alertSummaryItemTitle}>{dataKey}</div>
-      <div className={styles.alertSummaryItemContent} title={valueAsText}>{value}</div>
+      <div className={styles.alertSummaryItemContent} title={valueAsText}>{valueAsText}</div>
```

Comment:
is this needed?

Reply:
Yeah, valueAsText is needed. Value is being displayed in ui which carry node_id(image_id) and is used to trigger api.
valueAsText is used to display to only ui not use for any logical things.